<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCalculate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate It!</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineInput1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineInput2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblAnswer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAnswer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txIntroduction">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>Description:

Be Care: This program expect you input a vaild numbers.

This program is show that some simple Qt control and c++.  Using Combo Box and Line Box for Elementary Arithmetic.

Wish you have good coding day and if any feedback, just ask(my real friend) or find me on github(not real friend). :::)))</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboChoices">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>51</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>4</width>
      <height>4</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbCopyLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Len Chan, No CopyRight(Public Domain).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
